backend/src/handlers/utils.go

```diff
@@ -92,3 +120,80 @@ func (srv *Server) getQueryContext(r *http.Request) models.QueryContext {
 		All:        all,
 	}
 }
+
+type emailReq struct {
+	to       string
+	from     string
+	subject  string
+	bodyText string
+	bodyHTML string
+}
+
+func newEmailReq(to, from, subject, bodyText, bodyHTML string) *emailReq {
+	return &emailReq{
+		to:       to,
+		from:     from,
+		subject:  subject,
+		bodyText: bodyText,
+		bodyHTML: bodyHTML,
+	}
+}
+
+func newAllowlistRequest(user *models.User, link *models.HelpfulLink) *emailReq {
+	reqTime := time.Now().Format("2006-01-02 15:04:05")
+	emailStr := `A new Helpful Link has been added to the UnlockEd system by an Administrator.
```

Comment:
@PThorpe92 Should the message in the emailStr match the html version?

I modified the message for grammar. Can you have it say?

> An Administrator has added a new Helpful Link to the UnlockEd system and requests your review to confirm it is allow-listed for all users.